extensions/agent/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "agent",
   "displayName": "SQL Server Agent",
   "description": "Manage and troubleshoot SQL Server Agent jobs",
-  "version": "0.45.0",
+  "version": "0.46.0",
```

Comment:
Why are you bumping the version? Releasing a new version of the extension isn't going to fix this since the fix is in core code.

Reply:
Oops, I forgot this was in core